lib/connection/client.coffee

```diff
@@ -183,7 +183,7 @@ module.exports =
   notConnectedError: (action = 'do that') ->
     if not @isActive()
       atom.notifications.addError "Can't #{action} without a Julia client running.",
-        description: "Start Julia using Packages → Julia → Start Julia."
+        description: "Start Julia using **`Julia-Client: Start-Julia`** command."
```

Comment:
Should probably have both options here or keep this as is, since advanced users will know how to start Julia and everyone else is more likely to use the menus.

Reply:
Sure, I will follow the menus way :smile: